patches/chrome-browser-ui-webui-settings-metrics_reporting_handler.cc.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/browser/ui/webui/settings/metrics_reporting_handler.cc b/chrome/browser/ui/webui/settings/metrics_reporting_handler.cc
+index 7d8fb6922a81f4758c3cebbc0d06555647d56a66..dc13086ac5a5c66edf748900f6bcbfc68316b51f 100644
+--- a/chrome/browser/ui/webui/settings/metrics_reporting_handler.cc
```

Comment:
I think most or all of these can be removed by using chromium_src overrides. Just define `GOOGLE_CHROME_BUILD` include the original .cc file and then undefine it again.

Reply:
Definitely! Working on that now.

Reply:
We had discussion about this technique - https://github.com/brave/brave-core/pull/109#discussion_r197990521.
Defining `GOOGLE_CHROME_BUILD` on top of it, I think this can cause inconsistency

Reply:
given the number of patches we have like this, I think it might be better to selectively turn it off where we don't want it and can't override to avoid it